javascript/signin.js

```diff
@@ -0,0 +1,122 @@
+document.addEventListener('DOMContentLoaded', function () {
+  const form = document.querySelector('.sign-box');
+  const emailInput = document.querySelector('#signin-email');
+  const passwordInput = document.querySelector('#signin-password');
+  const errorTextEmail = document.createElement('div');
+  const errorTextPassword = document.createElement('div');
+  const passwordCover = document.querySelector('.eyecon');
+  let isPasswordCovered = false;
+
+  //////////////// 함수 정의 ////////////////////
+
+  // 이메일 확인 함수
+  function emailCheck() {
+    const email = emailInput.value.trim();
+    clearErrorStyle(emailInput);
+
+    // 기존 에러 메세지 삭제
+    if (errorTextEmail.parentNode === emailInput.parentNode) {
+      emailInput.parentNode.removeChild(errorTextEmail);
+    }
+    // 이메일 미입력 에러
+    if (email == '') {
+      showError(errorTextEmail, '이메일을 입력해 주세요.', emailInput);
+      return false;
+    }
+    // 이메일 형식 확인
+    const emailRegex = /^[^\s@]+@[^\s@]+\.[^\s@]+$/;
+    if (!emailRegex.test(email)) {
+      showError(errorTextEmail, '올바른 이메일 주소가 아닙니다.', emailInput);
+      return false;
+    }
+    return true;
+  }
+
+  // 비번 확인 함수
+  function passwordCheck() {
+    const password = passwordInput.value.trim();
+    clearErrorStyle(passwordInput);
+    // 기존 에러 메세지 삭제
+    if (errorTextPassword.parentNode === passwordInput.parentNode) {
+      passwordInput.parentNode.removeChild(errorTextPassword);
+    }
+    // 비밀번호 미입력 에러
+    if (password == '') {
+      showError(errorTextPassword, '비밀번호를 입력해 주세요.', passwordInput);
+      return false;
+    }
+    return true;
+  }
+
+  // 에러 스타일 빼기
+  function clearErrorStyle(inputElement) {
+    const parentContainer = inputElement.parentNode;
+    const errorElement = parentContainer.querySelector('.error-message');
+
+    parentContainer.classList.remove('error');
+    if (errorElement == true) {
+      parentContainer.removeChild(errorTextEmail);
+    }
+  }
+
+  // 에러 메세지 띄우기
+  function showError(errorText, message, inputElement) {
+    errorText.textContent = message;
+    errorText.classList.add('error-message');
+    inputElement.parentNode.appendChild(errorText);
+    inputElement.parentNode.classList.add('error');
+  }
+
+  // 눈 아이콘 변경하기
+  function togglePassword() {
+    const coverEye = '../image/icon/eye_closed.svg';
+    const openEye = '../image/icon/eye_open.svg';
+    if (isPasswordCovered) {
+      passwordCover.src = coverEye;
+    } else {
+      passwordCover.src = openEye;

```

Comment:
요런 간단한건 삼항연산자로도 처리할 수 있겠네요!
```suggestion
   passwordCover.src  = isPasswordCovered ? coverEye : openEye;
```

Reply:
할당할 값을 고르는데에도 삼항 연산자를 쓰는 방법이 잇었군요! 덕분에 깔끔하고 보기 쉬워졌습니다😊 c97a1e1b40c7e37083634ef31c99d77908939a79